backend/tests/unit_tests/config_definition/test_read.py

```diff
@@ -65,8 +65,8 @@ def _run_r_config_definition(
 
     @patch.object(
         DataStore,
-        "_execute_query",
-        return_value=[{"json_schema": '{"type": "object"}'}],
+        "execute_query",
+        return_value=(0, [{"json_schema": '{"type": "object"}'}]),
```

Comment:
@coderabbitai The reason why the `return_value` structure is not consistent, is because of the underlying queries. For many of the requests, and especially for the unit tests, it wouldn't make sense to keep `[{"json_schema": '{"type": "object"}'}]` in the response. I understand why you've brought this up, but I believe this is fine since unit tests aim for specificity. WDYT?